core/templates/domain/entity_editor_browser_tabs_info/entity-editor-browser-tabs-info.model.ts

```diff
@@ -26,11 +34,45 @@ export interface EntityEditorBrowserTabsInfoObject {
 }
 
 export class EntityEditorBrowserTabsInfo {
-  // Id of the entity whose editor page is opened.
+  /**
+    * It stores the type of the entity for a particular opened
+    * entity editor tab.
+    * It can have values: 'topic', 'story', 'skill' and 'exploration'.
+    * For example, if an editor tab with url '/topic_editor/topic_1' is
+    * opened, then this property will store 'topic'.
+   */
   _entityType: string;
+  /**
+   * It stores the id of the entity for a particular opened
+   * entity editor tab.
+   * For example, if an editor tab with url '/topic_editor/topic_1' is
+   * opened, then this property will store 'topic_1'.
+   */
   _id: string;
+  /**
+   * It stores the latest version for a particular entity with a particular id.
+   * For example, if an editor tab with url '/topic_editor/topic_1' is
+   * opened, then this property will store the
+   * latest version of the topic with id 'topic_1'.
```

Comment:
Not sure why the previous line got truncated so early?

Reply:
Fixed it now.